deploy/host-operator/e2e-tests/toolchainconfig.yaml

```diff
@@ -11,8 +11,9 @@ spec:
       durationBeforeNotificationDeletion: '5s'
     registrationService:
       analytics:
-        segmentWriteKey: 'test segment write key'
-        woopraDomain: 'test woopra domain'
+        devSpaces:
+          segmentWriteKey: 'test devspaces segment write key'
+        segmentWriteKey: 'test sandbox segment write key'
```

Comment:
just a reminder: Please don't forget to update the sandbox-sre resources when you merge these PRs